scripts/cloud/cloud-instance

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+
+CONFIG_LOCATION=$HOME/instructlab/cloud-config
```

Comment:
This will let you override the location by setting the `CONFIG_LOCATION` env var.

```suggestion
CONFIG_LOCATION=$(CONFIG_LOCATION:-$HOME/instructlab/cloud-config}
```

Reply:
Fixed, thanks!